dll/win32/kernel32/kernel32_vista/k32_vista.h

```diff
@@ -5,6 +5,7 @@
 #define WIN32_NO_STATUS
 #include <windef.h>
 #include <winbase.h>
+#include <strsafe.h>
```

Comment:
Please use `#include <ntstrsafe.h>` instead (see rest of kernel32 code, for example:
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/kernel32/k32.h;hb=0519ae0ac586d29e413b53c233a69e792d37a728#l48
) and do the necessary adaptations in your code below.

Reply:
OK.